core/src/test/java/nu/yona/server/entities/MockJpaRepositoryEntityWithUuid.java

```diff
@@ -27,7 +28,7 @@
 	}
 
 	@Override
-	public <S extends T> S findOne(Example<S> example)
+	public <S extends T> Optional<S> findOne(Example<S> example)
```

Comment:
`findById`?

Reply:
No, surprisingly. The interface ``QueryByExampleExecutor`` still has ``findOne``. Note that it is not 'by ID' but 'by example'.